src/js/factfinder/search-redirect.js

```diff
@@ -1,16 +1,19 @@
 document.addEventListener('ffReady', function () {
-    var redirectPath = window.ffRedirectPath || '/factfinder/result';
+     const redirectPath = window.ffRedirectPath || '/factfinder/result';
```

Comment:
Just had a look, there is no support for IE11. For the moment, unfortunately, i would stick to `var`

Reply:
const are supported by IE11
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

Older version could be problematic though, but not sure if still there are people using IE10 or 9